spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -32,34 +34,28 @@ describe('GOVUK.analyticsGa4.linkTracker', function () {
       links = document.createElement('div')
       links.innerHTML =
           '<div class="fully-structured-external-links">' +
-              '<a href="http://www.nationalarchives1.gov.uk"> National Archives </a>' +
-              '<a href="https://www.nationalarchives2.gov.uk"></a>' +
-              '<a href="https://www.nationalarchives3.gov.uk/one.pdf">National Archives PDF</a>' +
+              '<a href="http://www.nationalarchives.gov.uk/1" link_domain="http://www.nationalarchives.gov.uk"> National Archives </a>' +
+              '<a href="https://www.nationalarchives.gov.uk/2" link_domain="https://www.nationalarchives.gov.uk"></a>' +
+              '<a href="https://www.nationalarchives.gov.uk/3.pdf" link_domain="https://www.nationalarchives.gov.uk">National Archives PDF</a>' +
             '</div>' +
             '<div class="www-less-external-links">' +
-              '<a href="http://nationalarchives1.gov.uk"> National Archives </a>' +
-              '<a href="https://nationalarchives2.gov.uk"></a>' +
-              '<a href="https://nationalarchives3.gov.uk/one.pdf">National Archives PDF</a>' +
-            '</div>' +
-            '<div class="http-less-external-links">' +
-              '<a href="nationalarchives1.gov.uk"> National Archives </a>' +
-              '<a href="nationalarchives2.gov.uk"></a>' +
-              '<a href="nationalarchives3.gov.uk/one.pdf">National Archives PDF</a>' +
+              '<a href="http://nationalarchives.gov.uk" link_domain="http://nationalarchives.gov.uk"> National Archives </a>' +
+              '<a href="https://nationalarchives.gov.uk" link_domain="https://nationalarchives.gov.uk"></a>' +
+              '<a href="https://nationalarchives.gov.uk/one.pdf" link_domain="https://nationalarchives.gov.uk">National Archives PDF</a>' +
             '</div>' +
             '<div class="protocol-relative-external-links">' +
-              '<a href="//nationalarchives1.gov.uk"> National Archives </a>' +
-              '<a href="//nationalarchives2.gov.uk"></a>' +
-              '<a href="//nationalarchives3.gov.uk/one.pdf">National Archives PDF</a>' +
+              '<a href="//nationalarchives.gov.uk"> National Archives </a>' +
```

Comment:
The reason this test data was originally numbered was because during some debugging it was much easier to differentiate between the links if they had different domains - might be a feature worth keeping (or you could change the URL to have a unique identifier, as you did with the links on line 37 onwards).

Reply:
Can do, I simplified the domain so that I didn't have to manually add a `link_domain` attribute to each link, but I can add that in.